### DISCUSSION TITLE
Add a 'has conflicts' label to Godot's PR page

### DISCUSSION THREAD

**Discussion by catboy-catfish** [godotengine/godot-proposals#9453] [General Discussions]
# What does this discussion want you to do?

Add a 'has conflicts' label to Godot's pull request page to indicate when a pull req has file conflicts that must be resolved before merging successfully.

This is kind of in the same vein as 'breaks compat', 'needs testing' 'needs work' etc.

# Is this strictly necessary?

Obviously not. It's just more convenient for us to have 'has conflicts' as a pull req label.  This way we all won't have to scroll all the way down to the bottom of the page to find out if a pull req is mergeable or not.

**Comment by AThousandShips** [comment]
That would be a lot of work for maintainers to go through constantly, it's not difficult to just look at the bottom of the page, instead of adding a ton of work for maintainers who already have work as it is

If it lingers long with conflicts after being approved it'll be bumped, or fixed by a maintainer, or if needed tagged with `needs work`, which is applicable

**Reply by AThousandShips** [reply]
I maintain a bunch of open PRs, some of them large, and I run into conflicts relatively regularly, and I don't feel the need to have someone else do that work for me

What I would love was if GitHub had this feature, that'd help a lot

But for this tag to work it'd require a lot of constant attention from maintainers, if it isn't checked and re checked regularly it wouldn't be useful

Further tags don't ping anyone, so you'd still have to look, the only difference for *you* would be that you don't have to scroll down (or press the end key), but it'd mean a lot of work for a small team of active maintainers doing triage/tagging work

This, to me, is a prime example of work that's expected to be done by individual contributors, like by individual reporters of bugs (to do the testing, provide steps and MRPs), and of people making proposals (doing the work of research and providing solution ideas), not by maintainers, there needs to be a division of labor here

**Reply by catboy-catfish** [reply]
I see where you're coming from. I'm gonna close this thread now

**Comment by Calinou** [comment]
[Godot Team Reports](https://godotengine.github.io/godot-team-reports/) tracks which PRs are in a mergeable state based on the number of approvals and whether there are merge conflicts. This works without labels needing to be manually assigned.

**Reply by catboy-catfish** [reply]
Not to sound dismissive, but am seriously considering deleting this thread now. I've already closed it. That's cause I see where you and AThousandShips are coming from. The maintainers already have enough shit on their plate. I don't want to add more anymore.

Thank you and AThousandShips for participating.

**Reply by AThousandShips** [reply]
I'm sorry that further response to this is causing you any distress, but this is just additional information relevant to the issue

**Reply by catboy-catfish** [reply]
> I'm sorry that further response to this is causing you any distress, but this is just additional information relevant to the issue

All good
